### PR TITLE
added tests to check that student completed the readme

### PR DESCRIPTION
**Pull request by jjseymour**
The only thing I struggled with was testing the ability to add a new attribute then rollback the database. I left what I wanted to test commented out, maybe someone would have a nice way of going about this.

### CODE REVIEW

File: spec/artist_spec.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+    expect(Artist.all.last.name).to eq("Justice")
+  end
+
+  it "can find an Artist by name" do
+    weeknd = Artist.create(name: 'The Weeknd', genre: "Alternative R&B", age: Time.now.strftime("%Y").to_i - 2010, hometown: "Toronto")
+    taytay = Artist.create(name: 'Taylor Swift', genre: "Pop / Country", age: Time.now.strftime("%Y").to_i - 2006, hometown: "Reading")
+    queen = Artist.create(name: 'Queen', genre: "Rock", age: Time.now.strftime("%Y").to_i - 1973, hometown: "London")
+
+    expect(Artist.find_by(name: 'Taylor Swift').genre).to eq("Pop / Country")
```
Comment (pletcher) [inline spec/artist_spec.rb]: Could we change this to `expect(Artist.find_by(name: "Taylor Swift")).to eq(taytay)`?

File: spec/artist_spec.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  end
+
+  it "can roll back to have no gender attribute for Artist" do
```
Comment (pletcher) [inline spec/artist_spec.rb:38]: This test won't pass, right?

File: spec/artist_spec.rb (hunk 1 above)
Comment (jjseymour) [inline spec/artist_spec.rb]: No problem. Just fixed this test.

File: spec/artist_spec.rb (hunk 2 above)
Comment (jjseymour) [inline spec/artist_spec.rb:38]: My original plan was to some how create a test that could test a migration that added a column then roll it back. But that proved to be difficult. Instead the last spec test will pass even before they add the new gender column migration. So, it basically is just a check that they rollback the migration before the end. Unfortunately, they would also be able to pass all of the tests without ever creating the new column and rolling it back.

**Comment by pletcher** [comment]
Okay. I think we can roll with this, since it's not a "lab" per se.

Would love to figure out a clean way to test that rollback, though. There's something like [this](http://blog.carbonfive.com/2011/01/27/start-testing-your-migrations-right-now/), but that honestly feels a bit odd to do.

**Pull request merged**